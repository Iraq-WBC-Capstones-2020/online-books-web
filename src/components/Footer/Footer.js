import React from 'react';

const Footer = () => {
  return (
    <footer className="w-full">
      <svg
      className="h-auto"
        viewBox="0 0 1440 653"
        fill="none"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M345.185 157.161C146.86 151.813 32.8747 60.2452 0 0V653H1440V327.866C1420.4 333.321 1399.19 335.723 1376.26 334.517C1291.08 330.039 1273.15 212.168 1102.8 236.291C424.32 332.369 636.574 165.019 345.185 157.161Z"
          fill="#F9F9F9"
        />
      </svg>
    </footer>
  );
};

export default Footer;
