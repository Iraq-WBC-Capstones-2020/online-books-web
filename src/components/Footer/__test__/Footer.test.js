import React from 'react';
import renderer from 'react-test-renderer';
import Footer from './../Footer';
import { useTranslation } from 'react-i18next';
test('Footer snapshot', () => {
  useTranslation().t = jest.fn().mockImplementation((key) => {
    if (key === 'footer.tagsList') {
      return ['Shop', 'Books', 'Online', 'Read', 'Knowledge'];
    }
    return key;
  });
  const tree = renderer.create(<Footer />).toJSON();
  expect(tree).toMatchSnapshot();
});
