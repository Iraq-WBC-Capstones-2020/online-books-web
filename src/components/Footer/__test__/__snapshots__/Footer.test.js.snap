// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snapshot 1`] = `
<footer
  className="w-full -mt-10 md:-mt-20 lg:-mt-32"
>
  <svg
    className="h-auto"
    fill="none"
    viewBox="0 0 1440 403"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M345.185 157.161C146.86 151.813 32.8747 60.2452 0 0V403H1440V327.866C1420.4 333.321 1399.19 335.723 1376.26 334.517C1291.08 330.039 1273.15 212.168 1102.8 236.291C424.32 332.369 636.574 165.019 345.185 157.161Z"
      fill="#f7fafc"
    />
  </svg>
  <div
    className="bg-gray-100 pb-3"
  >
    <div
      className="container"
      data-test="container"
    >
      <div
        className="row"
        data-test="row"
      >
        <div
          className="col-xs-12 col-sm-6 col-md-4 col-lg-4"
          data-test="col"
        >
          <img
            alt="logo"
            className="h-20 d-block d-sm-inline mx-auto"
            src="https://via.placeholder.com/150x100"
          />
          <p
            className="mt-2 text-base leading-6 text-center text-sm-left"
          >
            footer.description
          </p>
          <div
            className="d-sm-none my-5 text-center"
          >
            <a
              href="https://www.linkedin.com"
            >
              <i
                className="fab fa-linkedin w-5"
                data-test="fa"
              />
            </a>
            <a
              href="https://www.facebook.com"
            >
              <i
                className="fab fa-facebook-f w-5"
                data-test="fa"
              />
            </a>
            <a
              href="https://www.twitter.com"
            >
              <i
                className="fab fa-twitter w-5"
                data-test="fa"
              />
            </a>
          </div>
          <div
            className="mt-5 text-sm font-bold text-center text-sm-left"
          >
            footer.copyright
          </div>
        </div>
        <div
          className="col-xs-0 col-sm-6 col-md-4 col-lg-4"
          data-test="col"
        >
          <div
            className="d-none d-sm-block"
          >
            <h4
              className="text-blue-500 font-bold mb-3 uppercase"
            >
              footer.followUs
            </h4>
            <ul
              className="text-xl leading-8"
            >
              <li>
                <a
                  href="https://www.linkedin.com"
                >
                  <i
                    className="fab fa-linkedin w-8"
                    data-test="fa"
                  />
                  LinkedIn
                </a>
              </li>
              <li>
                <a
                  href="https://www.facebook.com"
                >
                  <i
                    className="fab fa-facebook-f w-8"
                    data-test="fa"
                  />
                  Facebook
                </a>
              </li>
              <li>
                <a
                  href="https://www.twitter.com"
                >
                  <i
                    className="fab fa-twitter w-8"
                    data-test="fa"
                  />
                  Twitter
                </a>
              </li>
            </ul>
          </div>
        </div>
        <div
          className="col-0 col-md-4 col-lg-4"
          data-test="col"
        >
          <div
            className="d-none d-md-block"
          >
            <h4
              className="text-blue-500 font-bold mb-3 uppercase"
            >
              footer.tags
            </h4>
            <h4
              className="inline-block capitalize font-semibold mr-2 mb-2 py-2 px-4 border border-black rounded-full"
            >
              Shop
            </h4>
            <h4
              className="inline-block capitalize font-semibold mr-2 mb-2 py-2 px-4 border border-black rounded-full"
            >
              Books
            </h4>
            <h4
              className="inline-block capitalize font-semibold mr-2 mb-2 py-2 px-4 border border-black rounded-full"
            >
              Online
            </h4>
            <h4
              className="inline-block capitalize font-semibold mr-2 mb-2 py-2 px-4 border border-black rounded-full"
            >
              Read
            </h4>
            <h4
              className="inline-block capitalize font-semibold mr-2 mb-2 py-2 px-4 border border-black rounded-full"
            >
              Knowledge
            </h4>
          </div>
        </div>
      </div>
    </div>
  </div>
</footer>
`;
